dll/win32/browseui/shellfind/CFindFolder.cpp

```diff
@@ -213,14 +213,47 @@ static UINT SearchFile(LPCWSTR lpFilePath, _SearchData *pSearchData)
         return 0;
 
     UINT uMatches = 0;
-    // Check for UTF-16 BOM
-    if (size >= 2 && lpFileContent[0] == 0xFF && lpFileContent[1] == 0xFE)
+    if (size >= 2 &&
```

Comment:
See https://git.reactos.org/?p=reactos.git;a=blob;f=base/applications/cmdutils/more/more.c;h=161818a5e6ba6d0219e84e887d23467df5aabe9d;hb=HEAD#l149 for more details.

Reply:
Perfect encoding detection is expensive in this case.